eng/Versions.props

```diff
@@ -184,5 +184,7 @@
     <SystemTextJsonVersion>6.0.0</SystemTextJsonVersion>
     <XunitAssertVersion>$(XunitVersion)</XunitAssertVersion>
     <XunitExtensibilityExecutionVersion>$(XunitVersion)</XunitExtensibilityExecutionVersion>
+    <!-- Benchmarks -->
+    <Benchmarks_BaselineSourceGeneratorsVersion>7.0.0-preview.5.22528.1</Benchmarks_BaselineSourceGeneratorsVersion>
```

Comment:
Is this an existing pattern in arcade that we're feeding into or a new one we're craeting?

Reply:
We have a sort of pattern where we have things like `Tooling_` it's not an official arcede-ism, but these are manually updated anyway, so arcade doesn't know about them